hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm.js

```diff
@@ -32,6 +32,15 @@
             $scope.RecordsToDisplay = "10";
             $scope.currentPage = 1;
             $scope.lastIndex = 0;
+            $scope.statistics = {
+                nodes : {
+                    usages : {}
+                }
+            }
+            $scope.statistics.nodes.usages.min = "";
+            $scope.statistics.nodes.usages.max = "";
+            $scope.statistics.nodes.usages.median = "";
+            $scope.statistics.nodes.usages.stdev = "";
```

Comment:
Hi @jianghuazhu, was there any reason behind separating these values out?
I believe we can declare them in the statistics object.
Also I think initializing them to something like "N/A" would be better as in case there is any error, the default value won't be empty but rather show something to let the user know something went wrong.

```suggestion
            $scope.statistics = {
                nodes : {
                    usages : {
                      min: "N/A",
                      max: "N/A",
                      median: "N/A",
                      stdev: "N/A"
                    }
                }
            }
```